projects/control-service/projects/pipelines_control_service/src/test/java/com/vmware/taurus/service/JobImageDeployerTest.java

```diff
@@ -113,13 +113,15 @@ public void testScheduleLabelsAnnotations() {
             var deployedDate = annotations.get(JobAnnotation.DEPLOYED_DATE.getValue());
             var startedBy = annotations.get(JobAnnotation.STARTED_BY.getValue());
             var executionType = annotations.get(JobAnnotation.EXECUTION_TYPE.getValue());
+            var unscheduled = annotations.get(JobAnnotation.UNSCHEDULED.getValue());
 
             Assertions.assertEquals("schedule string", schedule);
             Assertions.assertEquals("lastDeployedBy", deployedBy);
             Assertions.assertEquals(OffsetDateTime.now().toString().substring(0, 10), deployedDate.toString().substring(0, 10),
                             "Testing if date is the same, we cannot be precise down to the millisecond.");
             Assertions.assertEquals("scheduled/runtime", startedBy);
             Assertions.assertEquals("scheduled", executionType);
+            Assertions.assertEquals("false", unscheduled);
```

Comment:
All k8s jobs are currently "com.vmware.taurus/execution-type: scheduled" annotated, correct? 
What are the annotation changes expected with this PR, maybe add to description?

Reply:
Jobs will remain annotated `execution-type: scheduled` as per Miro's comment higher up.